orchagent/portsorch.cpp

```diff
@@ -2737,6 +2738,29 @@ void PortsOrch::doPortTask(Consumer &consumer)
 
                 for (auto it = m_lanesAliasSpeedMap.begin(); it != m_lanesAliasSpeedMap.end();)
                 {
+                    bool lane_set_is_available = true;
+                    for (auto lane: it->first)
+                    {
+                        if (m_availableLanes.find(lane) == m_availableLanes.end())
+                        {
+                            lane_set_is_available = false;
+                            break;
+                        }
+                    }
+                    if (!lane_set_is_available)
+                    {
+                        string tmp_lane_str = "";
+                        for (auto s : it->first)
+                        {
+                            tmp_lane_str += to_string(s) + " ";
+                        }
+                        tmp_lane_str = tmp_lane_str.substr(0, tmp_lane_str.size() - 1);
+
+                        SWSS_LOG_WARN("The lane set %s isn't available in ASIC", tmp_lane_str.c_str());
```

Comment:
it looks like this conflicts with line 2764. in case the port is not available, the orchagent tends to create that port.

Reply:
I don't think so. This block is to check whether the expected lanes of ports are available. But line 2764 is to check whether a port is configured for creating.